### PR TITLE
Use IPython 3.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Hooray IPython 3.0 :cake:! Welcome to the future :monorail:!

This upgrades nbviewer's IPython requirement to `IPython[notebook]>=3.0`.

Since all good things come in threes, I'd love to see (IPython, Bootstrap & Reveal (ahem, #390)) 3.x all wrapped up as a 0.3.0 release!

### CODE REVIEW

**Comment by bollwyvl** [comment]
Error appears to be Travis->Github DNS...

**Comment by bollwyvl** [comment]
Hooray, build fixed. Docker build improved somewhat as well.... mainly to prevent re-pip/bower-ing after every file change for local test... though this could be taken out, if that's too far outside the scope.

**Comment by bollwyvl** [comment]
Gah, or the build is not fixed. Again, seeing lots fo `40x` from github. My oh my.

File: Dockerfile [hunk 1]
```diff
@@ -28,23 +28,27 @@ RUN npm install -g bower less
...
+ADD ./requirements.txt /srv/nbviewer/
 WORKDIR /srv/nbviewer
 
-RUN invoke bower
-RUN invoke less
-
 RUN pip install -r requirements.txt
 
+ADD ./tasks.py /srv/nbviewer/
+ADD ["/nbviewer/static/bower.json", "/nbviewer/static/.bowerrc", "/srv/nbviewer/nbviewer/static/"]
```
Comment (rgbkrk) [inline Dockerfile]: Well this is new to me. How does `ADD` work with a list?

File: README.md [hunk 2]
```diff
@@ -63,6 +63,18 @@ docker build -t nbviewer .
...
+
+The Notebook Viewer uses `memcache` in production. To locally try out this
+setup, a [docker-compose](https://docs.docker.com/compose/) configuration is
+provided to easily start/stop the `nbviewer` and `memcache` containers together:
+
+#### Run
+
+```shell
+docker-compose up
+```
```
Comment (rgbkrk) [inline README.md]: Thank you for doing this! I've been waiting on compose to release, had not realize it had shipped.

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,3 @@
-tornado>=3.1.1
```
Comment (rgbkrk) [inline requirements.txt]: Technically this makes sure our underlying server matches tornado. I couldn't imagine IPython notebook changing (like it did from twisted) again, but still.

**Comment by rgbkrk** [comment]
Looking good. If the docker build is going well, I say ship this. The only comment I made that I care about a response is the `ADD` with a list.

File: Dockerfile (hunk 1 above)
Comment (rgbkrk) [inline Dockerfile]: Ah, it's intended to be `["<src>"... "<dest>"]`. Should these be `./nbviewer/static/...`?

Comment (bollwyvl) [inline Dockerfile]: You're right! Will fix!

File: README.md (hunk 2 above)
Comment (bollwyvl) [inline README.md]: Yeah they were very sneaky. It was confusing for a while as they still were using the fig gh.

File: requirements.txt (hunk 3 above)
Comment (bollwyvl) [inline requirements.txt]: True. Will fix!

**Comment by bollwyvl** [comment]
Addressed above comments. Changed the links to bower stuff... was definitely a typo, though ADD will [only, ahem, add](http://stackoverflow.com/questions/27068596/how-to-include-files-outside-of-dockers-build-context) things inside the current build context/dir. But it looked weird, for sure, and the `./` is easier to understand!

Eyeing the shiny green button, but wouldn't mind some more :eyes:! In the meantime will start working up the bootstrap3 stuff!

**Comment by rgbkrk** [comment]
Looks great to me. Thanks @bollwyvl!

**Pull request merged**